packages/camera/camera_platform_interface/lib/src/platform_interface/camera_platform.dart

```diff
@@ -47,12 +47,26 @@ abstract class CameraPlatform extends PlatformInterface {
   }
 
   /// Creates an uninitialized camera instance and returns the cameraId.
+  /// Method will be deprecated. Use [createCameraWithSettings].
```

Comment:
You should add an empty space above this: https://dart.dev/guides/language/effective-dart/documentation#do-separate-the-first-sentence-of-a-doc-comment-into-its-own-paragraph

Reply:
Done.